### PR TITLE
[Jetsnack] Update to snapshot 7067732

### PR DESCRIPTION
**Pull request by nickbutcher**

### CODE REVIEW

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Filters.kt [hunk 1]
```diff
@@ -75,18 +76,20 @@ fun FilterChip(
     shape: Shape = MaterialTheme.shapes.small
 ) {
     val (selected, setSelected) = filter.enabled
-    val backgroundColor =
-        animate(if (selected) JetsnackTheme.colors.brand else JetsnackTheme.colors.uiBackground)
+    val backgroundColor by
+    animateAsState(if (selected) JetsnackTheme.colors.brand else JetsnackTheme.colors.uiBackground)
```
Comment (manuelvicnt) [inline Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Filters.kt:80]: Indentation is slightly off here. Indentation in L86 looks great though, maybe follow the same pattern?

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt [hunk 2]
```diff
@@ -20,10 +20,11 @@ import androidx.annotation.FloatRange
 import androidx.annotation.StringRes
 import androidx.compose.animation.AnimatedFloatModel
 import androidx.compose.animation.Crossfade
-import androidx.compose.animation.animate
+import androidx.compose.animation.animateAsState
 import androidx.compose.animation.animatedFloat
 import androidx.compose.animation.core.AnimationSpec
 import androidx.compose.animation.core.SpringSpec
+import androidx.compose.animation.core.animateAsState
```
Comment (manuelvicnt) [inline Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt:27]: Two `animateAsState` imports here. Which one to use? I assume it's the one at the top

**Review by manuelvicnt — COMMENTED** [review]
Added some minor comments, thanks!

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt (hunk 2 above)
Comment (nickbutcher) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt:27]: Both are needed, the first is for animating a `Float` and the later is for animating a `color`. Does seem strange that they're in different packages, will investigate.

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Filters.kt (hunk 1 above)
Comment (nickbutcher) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Filters.kt]: done

**Review by manuelvicnt — APPROVED** [review]

**Pull request merged**